frescobaldi_app/app.py

```diff
@@ -137,7 +137,7 @@ def instantiate():
     QApplication.setDesktopFileName(appinfo.desktop_file_name)
     QApplication.setOrganizationName(appinfo.name)
     QApplication.setOrganizationDomain(appinfo.domain)
-    qApp.focusChanged.connect(clear_selection)
+    # qApp.focusChanged.connect(clear_selection)
```

Comment:
question: Is this some forgotten debug change?

Reply:
Yes, I forgot to restore it. That line was commented out because it clears the selection and so Quick Insert doesn't work anymore over selection when firing action with the mouse, clicking on the buttons in the panel.